src/test/java/seedu/address/logic/parser/ViewCommandParserTest.java

```diff
@@ -9,26 +9,37 @@
 import org.junit.jupiter.api.Test;
 
 import seedu.address.logic.commands.ViewCommand;
+import seedu.address.model.student.Id;
+import seedu.address.model.student.IdPredicate;
 import seedu.address.model.student.NameContainsKeywordsPredicate;
 
-public class FindCommandParserTest {
+public class ViewCommandParserTest {
 
-    private FindCommandParser parser = new FindCommandParser();
+    private ViewCommandParser parser = new ViewCommandParser();
 
     @Test
     public void parse_emptyArg_throwsParseException() {
         assertParseFailure(parser, "     ", String.format(MESSAGE_INVALID_COMMAND_FORMAT, ViewCommand.MESSAGE_USAGE));
     }
 
     @Test
-    public void parse_validArgs_returnsFindCommand() {
+    public void parse_validNameArgs_returnsViewCommand() {
         // no leading and trailing whitespaces
         ViewCommand expectedViewCommand =
                 new ViewCommand(new NameContainsKeywordsPredicate(Arrays.asList("Alice", "Bob")));
-        assertParseSuccess(parser, "Alice Bob", expectedViewCommand);
+        assertParseSuccess(parser, " nm/Alice Bob", expectedViewCommand);
 
         // multiple whitespaces between keywords
-        assertParseSuccess(parser, " \n Alice \n \t Bob  \t", expectedViewCommand);
+        assertParseSuccess(parser, " nm/\n Alice \n \t Bob  \t", expectedViewCommand);
+    }
+
+    @Test
+    public void parse_validIdArgs_returnsViewCommand() {
+        ViewCommand expectedViewCommand =
+                new ViewCommand(new IdPredicate(new Id("123A")));
+        assertParseSuccess(parser, " id/123a", expectedViewCommand);
+
+        assertParseSuccess(parser, " id/ \t 123a \n", expectedViewCommand);
     }
 
```

Comment:
An additional test case is probably needed here to cover invalid name or id args passed in the view command parser. To take note in the next PR @minjiunn